conans/cli/commands/__init__.py

```diff
@@ -26,7 +26,7 @@ def default(self, o):
             return repr(o)
         elif isinstance(o, Remote):
             return o.name
-        raise TypeError("Don't know how to serialize a {} object".format(o.__class__))
+        return JSONEncoder.default(self, o)
```

Comment:
Should we manage any exceptions raised here?

Reply:
Not really, the ``JSONEncoder.defaul()`` implementation will do that ``raise`` internally